test/classes/delete-free/tests-from-design-overview/borrowed-arguments.chpl

```diff
@@ -0,0 +1,22 @@
+class MyClass {
+  var x:int;
+  proc init() {
+    writeln("in MyClass.init");
+  }
+  proc deinit() {
+    writeln("in MyClass.deinit");
+  }
+}
+
+var global: borrowed MyClass; // 'borrowed' optional here
+proc saveit(arg: borrowed MyClass) { // and here
+  global = arg; // Error! trying to store borrow from local 'x' into 'global'
```

Comment:
Should the comment say "local 'arg'"?

Reply:
In this case, it's referring to 'x' in the other routine (test); it's intentional, but at the moment 'arg' has the value of 'x'. So the other interpretation is not wrong.